src/components/ValidateCodeActionModal/ValidateCodeForm/BaseValidateCodeForm.tsx

```diff
@@ -126,12 +126,9 @@ function BaseValidateCodeForm({
     );
 
     useEffect(() => {
-        if (!validateError) {
-            return;
-        }
         clearError();
         // eslint-disable-next-line react-compiler/react-compiler, react-hooks/exhaustive-deps
```

Comment:
Don't think we need this eslint-disable line anymore.

Also can you describe what this change is doing?

Reply:
> The original logic will auto-clear errors when there's one

Hm yeah that original logic doesn't make sense to me so I think this change is fine. cc @hungvu193 

> I think without the eslint disable, the compiler will mark this as missing clearError deps, am I correct?

We can keep `clearError` in the dependencies. However we should add `useCallback` [here](https://github.com/Expensify/App/blob/12830ffa9ca05e95b70c4fd8221a0439d64c108e/src/pages/settings/Wallet/VerifyAccountPage.tsx#L80)